### DISCUSSION TITLE
How do I "alt-tab" to existing instance of application instead of opening a new one

### DISCUSSION THREAD

**Discussion by RedwanFox** [Flow-Launcher/Flow.Launcher#1916] [Q&A] [answered]
How do i reach such behavior:
1. I open application A
2. I open application B
3. I type application A in flow and flow makes already opened instance of A focused
?

This works right now for several applications, because its handled on app side.

**Comment by deefrawley** [comment, marked as answer]
Install the Window Walker plugin
